README.asciidoc

```diff
@@ -0,0 +1,398 @@
+= GibiBot
+
+== Team members
+
+* *Robert Valge* (Programming)
+* *Argo Aljand* (Programming)
+* *Marc Ander Etti* (Mechanics)
+* *Raimo Köidam* (Electronics)
+
+== Links
+
+*Fusion 360 link:* https://a360.co/3fKQEJi  
+
+== Programming
+=== Packages
+* math - used for calculating different math equasions for omni motion
+* random - to make random choices
+* enum - used to make a state machine for movemnt and colours
+* time - for different timers, for example if robot is in FIND_BALL state for 3 seconds then it changes the state
+* pyrealsense2 - camera's library
+* threading - to make second thread for referee commands client
+* json - string manipulation
+* websockets - for client and referee server connection
+* asyncio - for referee commands client 
+* serial - needed for coding usb communication
+* struct - for packing information that is sent to the motherboard
+* serial.tools.list_ports - to check for available usb ports, so we don't have to find the correct port's name manually
+* jit_functions - used it in image processing, jit makes for loops much faster so we can go through lists much faster
+* numpy - for list manipulation
+* cv2 - image processing
+* tkinter - used it in the beginning to simulate omnimotion visually
+* websockets - to make our testing server for referee commands
+
+=== A brief description of your game logic and overall code structure.
+
+Our code overall is section into files according to their usage (for example immage_processing.py and movement.py) to make the code less clutterd then if it was all in the same file. The fails contain mainly different classes and then we can import them according to our needs. The main code is structured so that every situatsion has a State that we can refere to to make the robot move accordingly.
+
+=== Block diagram of game logic
+image::https://user-images.githubusercontent.com/112713620/212192438-b3be92dc-7776-4a3d-9623-ac57508f8613.JPG[750,750]

```

Comment:
Add description to block diagram arrows and/or create condition blocks. Current diagram does not describe game logic.